VS2017/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM microsoft/windowsservercore:10.0.14393.1944
```

Comment:
Why is this Windows build number different from the one in Dockerfile.vs2015? Just because this file was created a few days before the other one? Or is there a more fundamental reason?

Reply:
This should be `.2007` -- good catch.